src/Microsoft.AspNetCore.Blazor.Browser.JS/Microsoft.AspNetCore.Blazor.Browser.JS.csproj

```diff
@@ -15,6 +15,11 @@
     <WebpackInputs Include="**\*.ts" Exclude="node_modules\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <WebpackInputs Remove="src\Interop\DotNetInvoker.ts" />
+    <WebpackInputs Remove="src\Interop\MethodOptions.ts" />
+  </ItemGroup>
```

Comment:
I suspect these changes aren't intended to be included. VS does this weird thing of auto-adding these items, which we then need to take back out manually.

Reply:
Yeah, VS being VS. I wanted to put the PR out before doing the cleanups, so don't worry about it. I'll clean these things out and webpackify all things typescript.